internal/v1/users.go

```diff
@@ -92,6 +92,23 @@ func (h *handler) QueryUsers(p httprequest.Params, r *params.QueryUsersRequest)
 	return usernames, nil
 }
 
+// RemoveUserMFACredentials removes all MFA credentials for a user.
+func (h *handler) RemoveUserMFACredentials(p httprequest.Params, r *params.RemoveUserMFACredentialsRequest) error {
```

Comment:
Would ClearUserMFACredentials be a better name?

Reply:
done